cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -155,12 +155,12 @@ var debug = log.New(ioutil.Discard, "", log.LstdFlags)
 // cores... 5m was not enough.
 const noOutputTimeout = 30 * time.Minute
 
-func (s *Server) loggedRun(tr trace.Trace, cmd *exec.Cmd) (err error) {
+func (s *Server) loggedRun(tr trace.Trace, description string, cmd *exec.Cmd) (err error) {
 	out := &synchronizedBuffer{}
 	cmd.Stdout = out
 	cmd.Stderr = out
 
-	tr.LazyPrintf("%s", cmd.Args)
+	tr.LazyPrintf("[%s]: %s", description, cmd.Args)
```

Comment:
is this the only place we put the description? This isn't that useful, since it just goes to `net/trace` which we don't view that often and the `tr` would already be linked to the purpose / the command args will say what it is doing.

Reply:
I'll back out this change then